client/sites-dashboard/components/test/sites-grid-item.tsx

```diff
@@ -38,38 +38,11 @@ describe( '<SitesGridItem>', () => {
 		} );
 	} );
 	test( 'Default render', () => {
-		const tree = renderer
-			.create(
-				<SitesGridItem site={ makeTestSite( { title: 'The example site' } ) as SiteExcerptData } />
-			)
-			.toJSON();
-		expect( tree ).toMatchSnapshot();
-	} );
-
-	test( 'Custom render', () => {
-		const tree = renderer
-			.create(
-				<SitesGridItem
-					site={ makeTestSite( { title: 'The example site' } ) as SiteExcerptData }
-					showLaunchNag={ false }
-					showBadgeSection={ false }
-					showThumbnailLink
-					showSiteRenewLink={ false }
-				/>
-			)
-			.toJSON();
-		expect( tree ).toMatchSnapshot();
-	} );
-
-	test( 'Custom render 2', () => {
 		const tree = renderer
 			.create(
 				<SitesGridItem
 					site={ makeTestSite( { title: 'The example site' } ) as SiteExcerptData }
-					showLaunchNag={ false }
-					showBadgeSection={ false }
-					showThumbnailLink={ false }
-					showSiteRenewLink={ false }
+					onSiteSelectBtnClick={ () => {} }
 				/>
 			)
 			.toJSON();
```

Comment:
This test feels a bit useless 😅 I'm not sure what it's even attempting to test.

Reply:
Agree, I'm just going to remove it I think :)

Reply:
Go for it 🧹